e2e/suites/convert-entity-template.test.ts

```diff
@@ -0,0 +1,62 @@
+/*global page*/
+
+import { ElementHandle } from 'puppeteer';
+import { createEntity } from '../helpers/createEntity';
+import { createTemplate } from '../helpers/createTemplate';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+import insertFixtures from '../helpers/insertFixtures';
+import disableTransitions from '../helpers/disableTransitions';
+
+const setupPreFlights = async (): Promise<void> => {
+  await insertFixtures();
+  await proxyMock();
+  await adminLogin();
+};
+
+const setupTest = async () => {
+  await createTemplate('Without image');
+  await createTemplate('With image');
+  await createEntity('Without image', {
+    pdf: `${__dirname}/test_files/valid.pdf`,
+    supportingFile: `${__dirname}/test_files/batman.jpg`,
+  });
+};
+
+describe('Convert entity template', () => {
+  beforeAll(async () => {
+    await setupPreFlights();
+    await setupTest();
+    await disableTransitions();
+  });
+
+  it('Should select image for image property from supporting files', async () => {
+    await expect(page).toClick('a[type="button"]');
+
+    await expect(page).toClick('.metadata-sidepanel button.edit-metadata', {
+      text: 'Edit',
+    });
+    await expect(page).toMatchElement('select.form-control > option');
+    const options = await page.$$('select.form-control > option');
+
+    const optionsValues = await options.map(async (optionElement: ElementHandle<Element>) => {
+      const value = await optionElement.evaluate(optionEl => ({
+        text: optionEl.textContent,
+        value: optionEl.getAttribute('value') as string,
+      }));
+      return value;
+    });
+
+    const optionValues: any[] = await Promise.all(optionsValues);
+    const { value } = optionValues.find(option => option.text === 'With image');
+
+    await expect(page).toMatchElement('select.form-control');
+    await page.select('select.form-control', value);
```

Comment:
Wasn't it possible to use the template with image right from the beginning? I think it would serve the purpose and save a lot of code. Maybe I'm missing something :D

Reply:
I also think that would help, the problem is I did not find any template that has an image out of all the ones I looked

Reply:
Yeah, but wouldn't it be useful to create the entity directly with template "With image" rather than start with "Without image" and then changing it to "With image"? My concern is: does that template changing add anything to the test?